package.json

```diff
@@ -81,6 +81,7 @@
     "sinon-chai": "^3.7.0",
     "stylelint": "^14.13.0",
     "svgo": "^2.8.0",
+    "swr": "^2.0.0",
```

Comment:
I would expect to see this as a dependency of the `document-capture` package, not at the root `package.json` ([docs](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#dependencies)).

```
yarn workspace @18f/identity-document-capture add swr
```

Reply:
@aduth I initially added `swr` as a dependency of only the `document-capture` package. As Matt alluded to, the tests ran and passed, but the build’s lint step broke on `yarn run lint`.  In order to get a passing build, I had to add `swr` as a dependency at the root level.

Reply:
Huh, I see what you mean now. That's a surprisingly tough issue to solve, and I'm surprised it hasn't bitten us before. Digging into it, I think it has to do with how [we configure `import/no-extraneous-dependencies`](https://github.com/18F/identity-idp/blob/a6d4e130b6802e5400d76cb025003518a13aaad8/.eslintrc#L35-L41) to allow the workspace package specs to use `devDependencies` defined in the root `package.json`. The default behavior of the rule would [find the workspace package's own dependencies](https://github.com/import-js/eslint-plugin-import/blob/766af5ff6c2e97cbe2a2a211729f7c129da3005d/src/rules/no-extraneous-dependencies.js#L81-L84), but the default and our override for root `devDependencies` apparently can't coexist to allow dependencies from the workspace + devDependencies in the root (which is the behavior we want).

I'd wonder if it'd be better to just turn off the rule for spec files (change [the existing override](https://github.com/18F/identity-idp/blob/a6d4e130b6802e5400d76cb025003518a13aaad8/.eslintrc#L35-L41) to `"import/no-extraneous-dependencies": "off"`). Otherwise, I think it could be okay to define in the root 'til we find a better solution to work around the rule's behavior.

Reply:
Good conversation!  I made a [jira ticket](https://cm-jira.usa.gov/browse/LG-8757) to find a more long-lasting solution.